{"impact": {"pr": null, "a": "COMPLETE", "ac": "LOW", "impact_score": null, "i": "COMPLETE", "cvss3_score": null, "c": "COMPLETE", "vector_string": "AV:L/AC:L/Au:N/C:P/I:N/A:N", "cvss2_severity": "LOW", "cvss3_severity": null, "s": null, "ui": null, "av": "LOCAL", "cvss2_score": 2.1, "au": "SINGLE", "exploitability_score": null}, "license": "Apache-2.0", "name": "python-keystoneclient", "cve_id": "CVE-2013-2013", "repository_summary": "Client Library for OpenStack Identity", "ochrona_severity_score": 2.1, "repo_url": "https://github.com/openstack/python-keystoneclient", "latest_version": "3.22.0", "cwe_id": "CWE-200", "language": "python", "publish_date": "2013-10-01T20:55Z", "references": ["http://www.openwall.com/lists/oss-security/2013/05/23/4", "https://bugs.launchpad.net/python-keystoneclient/+bug/938315", "https://oval.cisecurity.org/repository/search/definition/oval%3Aorg.mitre.oval%3Adef%3A16937"], "owner": "openstack", "affected_versions": [{"operator": "<=", "version_value": "0.2.3"}], "description": "The user-password-update command in python-keystoneclient before 0.2.4 accepts the new password in the --password argument, which allows local users to obtain sensitive information by listing the process."}